content/tutorial/01-simple-orders/02-app-data/02-create-order-app-data/README.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Create order with appData
+---
+
+Now, from the previous tutorial we have created a simple app data document. We will now use this document to create an order.
+
+In doing so, we will be making use of:
+
+- [Submit order](/tutorial/submit-order)
+- [Simple app data](/tutorial/simple-app-data)
+
+## Instantiate the SDK
+
+We will start from the basic setup from the [submit order](/tutorial/submit-order) tutorial and the [simple app data](/tutorial/simple-app-data) tutorial. This has been populated in the code editor for you.
+
+
+### Quoting with app data
+
+The keen eye-ed among you will notice that `appDataHex` and `appDataContent` are not used. Let's fix that. When we request a quote, we will pass `appDataHex` and `appDataContent` to the API. This allows the API to:
+
+- validate the app data document and it's has (`appDataHex`)
+- wrap the app data into the response object
+- determine any additional fees that may be required (if the app data document contains hooks)
+
+```typescript
+/// file: run.ts
+// ...
+export async function run(provider: Web3Provider): Promise<unknown> {
+    // ...
+
+    const quoteRequest: OrderQuoteRequest = {
+        sellToken,
+        buyToken,
+        from: ownerAddress,
+        receiver: ownerAddress,
+        sellAmountBeforeFee: sellAmount,
+        kind: OrderQuoteSideKindSell.SELL,
+        +++appData: appDataContent,+++
+        +++appDataHash: appDataHex,+++
+    };
+
+    // ...
+}
+```
+
+### Signing with app data
+
+When signing an order, we need to make sure that we have set the `appData` correctly. In this case, the `UnsignedOrder` used by the `OrderSigningUtils` class has an `appData` field which should be set to the `appDataHex` value.
+
+```typescript
+/// file: run.ts
+// ...
+export async function run(provider: Web3Provider): Promise<unknown> {
+    // ...
+    const order: UnsignedOrder = {
+        ...quote,
+        receiver: ownerAddress,
+        +++appData: appDataHex,+++
+    }
+    // ...
+}
+```
+
+## Run the code
+
+To run the code, we can press the "Run" button in the bottom right panel (the web container).
+
+When running the script, we may be asked to connect a wallet. We can use Rabby for this.
+
+1. Accept the connection request in Rabby
```

Comment:
General comment: should we be mentioning a specific wallet? What about to say instead: `in your wallet`

Reply:
At this stage, I think it's a little nicer to be specific. "in your wallet" may result in "why no wallet connect", "why no trezor wallet" etc (otherwise we would then have to say your EIP-1193 compatible wallet, which then confuses many others).